api-extractor/src/definitions/ApiItem.ts

```diff
@@ -58,24 +58,24 @@ export enum ApiItemKind {
 }
 
 /**
- * The state of resolving the ApiItem's doc comment references inside a recursive call to ApiItem.resolveReferences().
+ * The state of completing the ApiItem's doc comment references inside a recursive call to ApiItem.resolveReferences().
  */
-enum ResolveState {
+enum CompleteState {
```

Comment:
CompletionState?  CompletingState?

This word has kind of clunky inflections heheh.